### PR TITLE
Update docs on accessing instances 📖

### LINKED ISSUE #1638: docs/accessing-instances.md is out of date
/kind documentation

docs/accessing-instances.md is out of date and needs to be updated for both Session Manager-based access, and bastion access using the new clusterctl workflows.

/priority important-soon
/milestone v0.5.x

### PR DESCRIPTION
**Pull request by scottslowe**
**What this PR does / why we need it**:

This PR updates the `docs/accessing-instances.md` file with the following changes:

- New `clusterctl` commands and workflows are included
- AWS CLI commands are updated to account for changes in tag names and values
- Instructions for using AWS SSM are added

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

Fixes #1638

### CODE REVIEW

File: docs/accessing-instances.md [hunk 1]
```diff
@@ -2,82 +2,119 @@
...
+    enabled: true
 ```
 
-### Obtain public IP address of the bastion node
+#### Obtain public IP address of the bastion node
 
-> Your credentials must let you query the EC2 API.
+Once the workload cluster is up and running after being configured for an SSH bastion host, you can use this AWS CLI command to look up the public IP address of the bastion host (your credentials must let you query the EC2 API):
 
 ```bash
```
Comment (randomvariable) [inline docs/accessing-instances.md:39]: Replace the aws query (or move into a troubleshooting section) with:

```
$ kubectl get awscluster
NAME   CLUSTER   READY   VPC                     BASTION IP
test   test      true   vpc-0db2f1a88bc3813b7    1.1.1.1
```

File: docs/accessing-instances.md [hunk 2]
```diff
@@ -2,82 +2,119 @@
...
+export CLUSTER_SSH_KEY=$HOME/.ssh/cluster-api-provider-aws
+```
+
+#### Get private IP addresses of nodes in the cluster
 
-> Your credentials must let you query the EC2 API.
+To get the private IP addresses of nodes in the cluster (nodes may be control plane nodes or worker nodes), use this AWS CLI command (note that your credentials must let you query the EC2 API):
 
 ```bash
-for type in controlplane node
```
Comment (randomvariable) [inline docs/accessing-instances.md:63]: Same as above, using kubectl:
```
➜ kubectl get awsmachine
NAME                       CLUSTER   STATE     READY   INSTANCEID                    MACHINE
test-control-plane-hrv2k   test      running   true    aws:////i-0604143dfff8e5a5c   test-control-plane-27z2l
```

File: docs/accessing-instances.md [hunk 3]
```diff
@@ -2,82 +2,119 @@
...
+
+### Accessing nodes via AWS Session Manager
+
+All CAPA-published AMIs based on Ubuntu have the AWS SSM Agent pre-installed (as a Snap package; this was added in June 2018 to the base Ubuntu Server image for all 16.04 and later AMIs). This allows users to access cluster nodes directly, without the need for an SSH bastion host, using the AWS CLI and the Session Manager plugin.
+
+To access a cluster node (control plane node or worker node), you'll need the instance ID. You can retrieve the instance ID using this command:
+
+```bash
+for type in control-plane node
+do
```
Comment (randomvariable) [inline docs/accessing-instances.md:104]: Same as above, using kubectl:
```
➜ kubectl get awsmachine
NAME                       CLUSTER   STATE     READY   INSTANCEID                    MACHINE
test-control-plane-hrv2k   test      running   true    aws:////i-0604143dfff8e5a5c   test-control-plane-27z2l
```

File: docs/accessing-instances.md (hunk 1 above)
Comment (scottslowe) [reply docs/accessing-instances.md:39]: Resolved via a1e4900f.

File: docs/accessing-instances.md (hunk 2 above)
Comment (scottslowe) [reply docs/accessing-instances.md:63]: Fixed via a1e4900f.

File: docs/accessing-instances.md (hunk 3 above)
Comment (scottslowe) [reply docs/accessing-instances.md:139]: Corrected via a1e4900f.

**Comment by scottslowe** [comment]
@randomvariable I've incorporated your feedback in the latest commit (a1e4900f). PTAL and let me know if this is acceptable.

**Pull request merged**